### PR TITLE
Add JSON schema

### PR DESCRIPTION
**Pull request by spesnova**
## WHAT
- Add `prmd`
- Define schema for `build` resource
- Prepare API doc: https://github.com/wantedly/risu/blob/spesnova/json-schema/docs/api-v1-alpha.md
- Create build script: `script/build`
## WHY

To share the schema
## REF
- [Platform API Reference | Heroku Dev Center](https://devcenter.heroku.com/articles/platform-api-reference)

### CODE REVIEW

**Comment by spesnova** [comment]
@wantedly/infrastructure plz review this :pig:

File: schema/schema.json [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+          "type": [
+            "string"
+          ]
+        },
+        "dockerfile": {
+          "description": "path within the build context to the Dockerfile",
+          "example": "Dockerfile.dev",
+          "type": [
+            "string"
+          ]
```
Comment (awakia) [inline schema/schema.json:63]:
```
default: "Dockerfile"
```

とか入れると良さそう

Comment (spesnova) [inline schema/schema.json:63]: 入れた！
`default` parameter を初めて知った

ref: [JSON Hyper-Schema: Hypertext definitions for JSON Schema](http://json-schema.org/latest/json-schema-hypermedia.html)

**Comment by koudaiii** [comment]
LGTM

**Pull request merged**